src/config/config.js

```diff
@@ -190,5 +190,24 @@ export default {
    * @memberof module:config
    * @property {Array} known_notations
    */
-  known_notations: ["preferred-email-encoding@pgp.com", "pka-address@gnupg.org"]
+  known_notations: ["preferred-email-encoding@pgp.com", "pka-address@gnupg.org"],
+  /**
+   * use external indutny/elliptic library
+   */
+  external_indutny_elliptic: false,
+  /*
+   * use indutny/elliptic library
+   */
+  use_indutny_elliptic: true,
```

Comment:
Even more nitpicky but I would swap these two options around, to make the order more logical (first we check whether to use it, then we check where to find it).

Reply:
fixed